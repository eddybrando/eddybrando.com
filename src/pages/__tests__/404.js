import React from "react"
import ShallowRenderer from "react-test-renderer/shallow"

import NotFoundPage from "../404"

describe("NotFoundPage", () => {
  it("renders correctly", () => {
    const renderer = new ShallowRenderer()
    const result = renderer.render(<NotFoundPage />)
    expect(result).toMatchSnapshot()
  })
})
