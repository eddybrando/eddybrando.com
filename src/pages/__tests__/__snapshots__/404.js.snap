// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage renders correctly 1`] = `
<Layout
  headerTitle={
    Object {
      "append": ", 404 over here!",
      "highlight": "Oops",
      "prepend": "",
    }
  }
  mainContainerClass=""
  nav={Array []}
>
  <SEO
    description=""
    lang="en"
    meta={Array []}
    noTitleTemplate={false}
    title="404: Not found"
  />
  <h1>
    NOT FOUND
  </h1>
  <p>
    You just hit a route that doesn't exist... the sadness.
  </p>
</Layout>
`;
